### PR TITLE
DOC-1700 - removed references to Bintray back to 4.3, updated links, …

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
…fixed text and for 4.10, fixed other issues

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/archived-docs/corda-os/4.4/generating-a-node.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ title: Creating nodes locally
 A node can be created manually by creating a folder that contains the following items:
 
 
-* The Corda JAR, downloaded from [https://r3.bintray.com/corda/net/corda/corda/](https://software.r3.com:443/artifactory/corda-releases/net/corda/corda/4.4) (under `/4.4/corda-4.4.jar`)
+* The Corda JAR, downloaded from https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.4/corda-4.4.jar.
 * A node configuration file entitled `node.conf`, configured as per [Node configuration](corda-configuration-file.md)
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.4/generating-a-node.md:29]:
```suggestion
* A node configuration file entitled `node.conf`, configured as per [Node configuration](corda-configuration-file.md).
```

File: content/en/archived-docs/corda-os/4.4/generating-a-node.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ title: Creating nodes locally
...
 * A folder entitled `cordapps` containing any CorDapp JARs you want the node to load
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.4/generating-a-node.md:30]:
```suggestion
* A folder entitled `cordapps` containing any CorDapp JARs you want the node to load.
```

File: content/en/archived-docs/corda-os/4.4/generating-a-node.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ title: Creating nodes locally
...
 * An up-to-date version of the <a href="network-map.html#network-parameters">`network-parameters` file</a> generated by the bootstrapper tool
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.4/generating-a-node.md:31]:
```suggestion
* An up-to-date version of the <a href="network-map.html#network-parameters">`network-parameters` file</a> generated by the bootstrapper tool.
```

File: content/en/archived-docs/corda-os/4.5/deploying-a-node.md [hunk 4]
```diff
@@ -50,9 +50,8 @@ handling, and ensures the Corda service is run at boot.
 `sudo adduser --system --no-create-home --group corda`
 
 * Create a directory called `/opt/corda` and change its ownership to the user you want to use to run Corda:`mkdir /opt/corda; chown corda:corda /opt/corda`
-* Download the [Corda jar](https://r3.bintray.com/corda/net/corda/corda/)
-(under `/4.5/corda-4.5.jar`) and place it in `/opt/corda`
-* Create a directory called `cordapps` in `/opt/corda` and save your CorDapp jar file to it. Alternatively, download one of
+* Download the Corda JAR file from https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.5/corda-4.5.jar and place it in `/opt/corda`.
+* Create a directory called `cordapps` in `/opt/corda` and save your CorDapp JAR file to it. Alternatively, download one of
 our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.5/deploying-a-node.md:55]:
```suggestion
our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory.
```

File: content/en/archived-docs/corda-os/4.7/deploying-a-node.md [hunk 5]
```diff
@@ -51,8 +51,7 @@ handling, and ensures the Corda service is run at boot.
 `sudo adduser --system --no-create-home --group corda`
 
 * Create a directory called `/opt/corda` and change its ownership to the user you want to use to run Corda:`mkdir /opt/corda; chown corda:corda /opt/corda`
-* Download the [Corda jar](https://r3.bintray.com/corda/net/corda/corda/)
-(under `/4.7/corda-4.7.jar`) and place it in `/opt/corda`
+* Download the Corda jar from https://software.r3.com/artifactory/corda-releases/net/corda/corda/4.7/corda-4.7.jar and place it in `/opt/corda`.
 * Create a directory called `cordapps` in `/opt/corda` and save your CorDapp jar file to it. Alternatively, download one of
 our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.7/deploying-a-node.md:56]:
```suggestion
our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory.
```

File: content/en/archived-docs/corda-os/4.7/deploying-a-node.md [hunk 6]
```diff
@@ -193,24 +192,14 @@ We recommend running Corda as a Windows service. This provides service handling,
...
-
-```shell
-mkdir C:\Corda
-wget http://jcenter.bintray.com/net/corda/corda/4.7/corda-4.7.jar -OutFile C:\Corda\corda.jar
-```
-
-
-* Create a directory called `cordapps` in `C:\Corda\` and save your CorDapp jar file to it. Alternatively,
-download one of our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory
+A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md)
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.7/deploying-a-node.md:195]:
```suggestion
A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md).
```

File: content/en/archived-docs/corda-os/4.8/deploying-a-node.md [hunk 7]
```diff
@@ -45,8 +45,6 @@ handling, and ensures the Corda service is run at boot.
 
 * A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md)
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.8/deploying-a-node.md:46]:
```suggestion
* A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md).
```

File: content/en/archived-docs/corda-os/4.8/deploying-a-node.md [hunk 8]
```diff
@@ -194,22 +192,14 @@ at boot, and means the Corda service stays running with no users connected to th
 
 **Prerequisites**:
 
-
-
 * A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md)
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.8/deploying-a-node.md:195]:
```suggestion
* A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md).
```

File: content/en/archived-docs/corda-os/4.8/deploying-a-node.md [hunk 9]
```diff
@@ -194,22 +192,14 @@ at boot, and means the Corda service stays running with no users connected to th
...
-
-* Create a directory called `cordapps` in `C:\Corda\` and save your CorDapp jar file to it. Alternatively,
+* Create a `Corda\cordapps` directory:
+   ```shell
+   mkdir C:\Corda\cordapps
+   ```
+* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.8/corda-4.8.jar
+* Copy the file to the Corda/cordapps directory, and rename it corda.jar.
+* Alternatively,
 download one of our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.8/deploying-a-node.md:203]:
```suggestion
download one of our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory.
```

File: content/en/platform/corda/4.10/community/deploying-a-node.md [hunk 10]
```diff
@@ -192,95 +192,74 @@ SystemD or Upstart configuration files so they are unique.
...
-* Run `services.msc` and verify that a service called `cordanode1` is present and running
-* Run `netstat -ano` and check for the ports you configured in `node.conf`
-* You may need to open the ports on the Windows firewall
-
-
-
+11. Enter your node’s desired legal name (see [Node identity](node-naming.html#node-identity) for more details).
+12. If required, add RPC users.
+13. Copy the required Java keystores to the node. See [Network certificates](permissioning.md)
+14. Download the [NSSM service manager](https://nssm.cc/)
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.10/community/deploying-a-node.md:237]:
```suggestion
14. Download the [NSSM service manager](https://nssm.cc/).
```

File: content/en/platform/corda/4.10/community/deploying-a-node.md [hunk 11]
```diff
@@ -192,95 +192,74 @@ SystemD or Upstart configuration files so they are unique.
...
+15. Unzip `nssm-2.24\win64\nssm.exe` to `C:\Corda`
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.10/community/deploying-a-node.md:238]:
```suggestion
15. Unzip `nssm-2.24\win64\nssm.exe` to `C:\Corda`.
```

File: content/en/platform/corda/4.9/community/deploying-a-node.md [hunk 12]
```diff
@@ -193,24 +193,14 @@ We recommend running Corda as a Windows service. This provides service handling,
...
-
-```shell
-mkdir C:\Corda
-wget http://jcenter.bintray.com/net/corda/corda/4.9/corda-4.9.jar -OutFile C:\Corda\corda.jar
-```
-
-
-* Create a directory called `cordapps` in `C:\Corda\` and save your CorDapp jar file to it. Alternatively,
-download one of our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory
+ A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md)
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.9/community/deploying-a-node.md:196]:
```suggestion
 A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md).
```

File: content/en/archived-docs/corda-os/4.4/deploying-a-node.md [hunk 13]
```diff
@@ -196,23 +196,15 @@ at boot, and means the Corda service stays running with no users connected to th
 
 **Prerequisites**:
 
+A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md)
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.4/deploying-a-node.md:199]:
```suggestion
A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md).
```
Punctuation is a bit inconsistent in those steps. I added full stops but feel free to ignore my comments if you want to have the steps without closing full stops. Just review them to make everything consistent please.

File: content/en/archived-docs/corda-os/4.4/deploying-a-node.md [hunk 14]
```diff
@@ -196,23 +196,15 @@ at boot, and means the Corda service stays running with no users connected to th
...
+* Create a `Corda\cordapps` directory:
+   ```shell
+   mkdir C:\Corda\cordapps
+   ```
+* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.4/corda-4.4.jar
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.4/deploying-a-node.md:204]:
```suggestion
* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.4/corda-4.4.jar.
```

File: content/en/archived-docs/corda-os/4.6/deploying-a-node.md [hunk 15]
```diff
@@ -192,25 +192,16 @@ SystemD or Upstart configuration files so they are unique.
...
-* Create a directory called `cordapps` in `C:\Corda\` and save your CorDapp jar file to it. Alternatively,
-download one of our [sample CorDapps](https://www.corda.net/samples/) to the `cordapps` directory
+**Prerequisites**
+
+A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md)
+* Create a `Corda\cordapps` directory:
+   ```shell
+   mkdir C:\Corda\cordapps
+   ```
+* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.6/corda-4.6.jar
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.6/deploying-a-node.md:202]:
```suggestion
* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.6/corda-4.6.jar.
```

File: content/en/archived-docs/corda-os/4.7/deploying-a-node.md [hunk 16]
```diff
@@ -193,24 +192,14 @@ We recommend running Corda as a Windows service. This provides service handling,
...
+* Create a `Corda\cordapps` directory:
+   ```shell
+   mkdir C:\Corda\cordapps
+   ```
+* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.7/corda-4.7.jar
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.7/deploying-a-node.md:200]:
```suggestion
* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.7/corda-4.7.jar.
```

File: content/en/archived-docs/corda-os/4.8/deploying-a-node.md [hunk 17]
```diff
@@ -194,22 +192,14 @@ at boot, and means the Corda service stays running with no users connected to th
...
-wget http://jcenter.bintray.com/net/corda/corda/4.8/corda-4.8.jar -OutFile C:\Corda\corda.jar
-```
-
-
-* Create a directory called `cordapps` in `C:\Corda\` and save your CorDapp jar file to it. Alternatively,
+* Create a `Corda\cordapps` directory:
+   ```shell
+   mkdir C:\Corda\cordapps
+   ```
+* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.8/corda-4.8.jar
```
Comment (AnnaK-R3) [inline content/en/archived-docs/corda-os/4.8/deploying-a-node.md:200]:
```suggestion
* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.8/corda-4.8.jar.
```

File: content/en/platform/corda/4.10/community/deploying-a-node.md [hunk 18]
```diff
@@ -192,95 +192,74 @@ SystemD or Upstart configuration files so they are unique.
...
-custom { jvmArgs = [ '-Xmx2048m', '-XX:+UseG1GC' ] }
-```
-
+1. Install a supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md).
+
+2. Create a `Corda\cordapps` directory:
+   ```shell
+   mkdir C:\Corda\cordapps
+   ```
+3. Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.10/corda-4.10.jar
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.10/community/deploying-a-node.md:201]:
```suggestion
3. Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.10/corda-4.10.jar.
```

File: content/en/platform/corda/4.9/community/deploying-a-node.md [hunk 19]
```diff
@@ -193,24 +193,14 @@ We recommend running Corda as a Windows service. This provides service handling,
...
+* Create a `Corda\cordapps` directory:
+   ```shell 
+   mkdir C:\Corda\cordapps
+   ```
+* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.9/corda-4.9.jar
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.9/community/deploying-a-node.md:201]:
```suggestion
* Download the following JAR file: https://software.r3.com/ui/native/corda-releases/net/corda/corda/4.9/corda-4.9.jar.
```

File: content/en/archived-docs/corda-os/4.4/deploying-a-node.md (hunk 13 above)
Comment (paulmoloneyr3) [reply content/en/archived-docs/corda-os/4.4/deploying-a-node.md:199]: Done - I'll add full stops where possible - for the lines ending in a URL, I'll move the URL to an indented separate line as the full stop looks odd at the end of it.

File: content/en/archived-docs/corda-os/4.4/deploying-a-node.md (hunk 14 above)
Comment (paulmoloneyr3) [reply content/en/archived-docs/corda-os/4.4/deploying-a-node.md:204]: Done.

File: content/en/archived-docs/corda-os/4.4/generating-a-node.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/archived-docs/corda-os/4.4/generating-a-node.md:30]: Done

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**